demos/embeddings/graphwave-barbell.ipynb

```diff
@@ -109,11 +113,11 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "### Visualation\n",
+    "### Visualisation\n",
     "\n",
     "The nodes are coloured by their structural role, e.g. in the fully connected sections, 1st node in the chain, 2nd node in the chain etc. We can see that all nodes of the same colour completely overlap in this visualation, indicating that structurally equivalent nodes are very close in the embedding space.\n",
     "\n",
-    "The plot here doesn't exactly match the one in the paper, which we think is because the details of approximating the wavelet diffusion differ (the paper uses Chebyshev polynomials, which isn't implemented in StellarGraph yet [#853](https://github.com/stellargraph/stellargraph/issues/853))."
+    "The plot here doesn't exactly match the one in the paper, which we think is because the details of approximating the wavelet diffusion differ, the paper uses `pygsp` to calculate the Chebyshev coefficient while `StellarGraph` uses `numpy` to calculate the coefficients. Some brief experiments have shown that the `numpy` Chebyshev coefficients are more accurate than the `pygsp` coefficients."
```

Comment:
Its pretty easy to add in, I've already tried `pysg` with `test_chebyshev` and it failed that test. also the embeddings were visually less well separated. 

It looks like `pysyg` compute the coefficients themselves here using a Riemann sum to do the integration: https://github.com/epfl-lts2/pygsp/blob/d78095438a14a74d2195d4600fc00f8f1115005f/pygsp/filters/approximations.py#L12

Reply:
I'm curious to validate the assertion here; that is, can we do a temporary modification and see if the resulting plot is closer to the paper?

Reply:
I can't get duplicate the papers results, even with the papers code! I suspect that its because I can't find the right hyperparameters.

But, there's another factor at play here. So, using the coefficients from `pysyg` I get the same results as the paper code, but `pysyg` have a bug and their coefficients are off by a factor of pi: https://github.com/epfl-lts2/pygsp/blob/d78095438a14a74d2195d4600fc00f8f1115005f/pygsp/filters/approximations.py#L52 vs Eq 8 https://arxiv.org/pdf/1105.1891.pdf

I might file an issue/PR with `pysyg`. Glad I found this, this was driving me crazy!

Reply:
> So, using the coefficients from pysyg I get the same results as the paper code

As in, the plot matches? Cool; that's all I was wondering.

> I might file an issue/PR with pysyg

Yeah, at least an issue would be great 👍

Reply:
Yeah I filed an issue; and I'll make a PR later :)

Reply:
https://github.com/epfl-lts2/pygsp/issues/70 for future reference.